internal/release/services/release/release.go

```diff
@@ -1,24 +1,84 @@
 package release
 
 import (
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
 	releaseSvc "github.com/terrariumcloud/terrarium/internal/release/services"
+	"github.com/terrariumcloud/terrarium/internal/storage"
+	"github.com/terrariumcloud/terrarium/pkg/terrarium/release"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	DefaultReleaseTableName = "terrarium-module-releases"
```

Comment:
The name of the table is wrong it should be terrarium-releases as we are going to use release for many things.